src/helm.bash

```diff
@@ -24,18 +25,19 @@ helm_template() {
   output_file="${tmp_write_dir}/templates.yaml"
 
   # NOTE: eval is used due to helm not liking empty template_opts
-  eval "helm template ${template_opts} ${chart_dir} &> ${output_file}"
+  return_message=$(eval "helm template ${template_opts} ${chart_dir} &> ${output_file}")
 
-  # Safety check to make sure nothing above silently failed
-  if [[ ! -s "${output_file}" ]] || [[ $(wc -c "${output_file}" | awk '{print $1}') -le 1 ]] ; then
-    fail "# FATAL-ERROR: (helm.bash): File is empty: '${output_file}'" || return $?
```

Comment:
Is it no longer necessary to check that `helm template` actually produces output?
If not, then I'd suggest getting rid of the `output_file` in this function all together as I don't see it being used for anything.

Reply:
`output_file` is still used for the helm.bash test itself, better to keep that

Reply:
ah yes, I see what you're saying. All good by me.